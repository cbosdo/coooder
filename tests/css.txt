.xformsItem 
{ 
/*  border: 1px dashed blue; */ 
    margin-right: 2px; 
    clear: both; 
    aqua: toto;
}
