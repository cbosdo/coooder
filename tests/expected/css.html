<pre class="css css" style="font-family:monospace;">.xformsItem 
<span style="color: #00AA00;">&#123;</span> 
<span style="color: #808080; font-style: italic;">/*  border: 1px dashed blue; */</span> 
    <span style="color: #000000; font-weight: bold;">margin-right</span><span style="color: #00AA00;">:</span> <span style="color: #933;">2px</span>; 
    <span style="color: #000000; font-weight: bold;">clear</span><span style="color: #00AA00;">:</span> <span style="color: #993333;">both</span>; 
    <span style="color: #000000; font-weight: bold;">aqua</span><span style="color: #00AA00;">:</span> toto;
<span style="color: #00AA00;">&#125;</span>
&nbsp;</pre>

